main.py

```diff
@@ -1,10 +1,53 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file_out:
+        players_data = json.load(file_out)
+
+    for nickname_, info_ in players_data.items():
+        email_ = info_["email"]
+        bio_ = info_["bio"]
+        guild_ = None
+        race_ = None
+
+        if info_["guild"]:
+            if not Guild.objects.filter(name=info_["guild"]["name"]).exists():
+                guild_ = Guild.objects.create(
+                    name=info_["guild"]["name"],
+                    description=info_["guild"]["description"]
+                )
+            else:
+                guild_ = Guild.objects.get(name=info_["guild"]["name"])
```

Comment:
You can use get_or_create
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create